src/main/webapp/app/exercises/text/assess/text-assessments.service.ts

```diff
@@ -167,4 +175,32 @@ export class TextAssessmentsService {
                 ),
         );
     }
+
+    /**
+     * Track the change of the Feedback in Athene
+     *
+     * @param submission - The submission object that holds the data that is tracked
+     */
+    public trackFeedback(submission: TextSubmission | null) {
+        // clone submission and resolve circular json properties
+        const submissionForSending = cloneDeep(submission);
+        submissionForSending!.participation.submissions = [];
+        submissionForSending!.participation.results.map((result) => {
+            result.participation = null;
+            result.submission = null;
+        });
+
+        const sendObject = {
+            textBlocks: submissionForSending?.blocks,
+            participation: submissionForSending?.participation,
+        };
+
+        if (submission!.participation?.trackingToken) {
+            this.http
+                .post(`${SERVER_API_URL}/athene-tracking/text-exercise-assessment}`, sendObject, {
```

Comment:
This endpoint does not exist on Artemis. Would it make sense to send this to Athene directly?

Reply:
The idea is to route it via nginx directly to athene. I've added an inline comment to make this clear for every developer.